### PR TITLE
Discussing choice of sequences in S57

### PR DESCRIPTION
**Pull request by StevenClontz**

### CODE REVIEW

**Comment by prabau** [comment]
Interesting question in mo.  So as specified in _Counterexamples_, i.e. before your change, the example was really a class of examples with at least $2^\mathfrak{c}$ homeomorphism classes between them.

But the same happens with the Single ultrafilter topology (S111).  Choosing a specific ultrafilter or specific converging sequences for this example determines the topology and different choices may give non-homeomorphic results.  We should definitely mention that.  However the topological properties discussed here are agnostic to these choices.  I feel that being too specific (like choosing the truncation of the decimal expansion of the irrational numbers) is a little bit misleading as this particular choice is never used specifically to prove any property.  Maybe we can leave it out until the day that a particular future property needs a further level of detail to differentiate things?

Also a nitpick: "Specifying ... necessary to characterize a unique topology".  Replacing a sequence with one with the same tail filter would give the same thing.  So maybe rephrase in a simpler way without using the word "necessary".

**Comment by prabau** [comment]
Actually that could be a good topic to add to the agenda for our next meeting: discussion of generic examples of spaces (i.e. not fully determined up to homeomorphism).

**Comment by lyengulalp** [comment]
I think this would be a good discussion. Psi-spaces also come to mind.

**Comment by StevenClontz** [comment]
As of 9612daf I've removed the specification of sequences, and merely note that this choice could be important someday - upon reflection I think it makes sense to only add in such specificity once we have a property that actually depends upon it. I've added this to be discussed at a future meeting.

**Review by prabau — APPROVED** [review]
